test/main.js

```diff
@@ -28,6 +28,7 @@ before('initialization', function () {
 	}
 	require('./../lib/process-manager').disabled = true;
 
+	config = Object.assign(config, require('../config/config-example'));
```

Comment:
Is the original loading still necessary?

Reply:
Is this modifying the `require` cache for config.js? If it is, it's still necessary so that you edit the cache for the right file.

Reply:
Oh, I think it might be.

Reply:
```suggestion
	Object.assign(config, require('../config/config-example'));
```

In which case, we should probably stop writing it in a way that suggests it doesn't have side effects.